src/controllers/event/event.controller.ts

```diff
@@ -2,19 +2,19 @@ import { Router, Request, Response, NextFunction } from "express";
 import eventModel from "./event.model";
 import Controller from "@interfaces/controller.interface";
 import { EventNotFoundException } from "@utils/exceptions/application.exception";
+import { title } from "process";
 
 class EventController implements Controller {
   public path = "/event";
   public router = Router();
   private event = eventModel;
-
   constructor() {
     this.initializeRoutes();
   }
 
   private initializeRoutes() {
     this.router.get(this.path, this.getAllEvents);
-    this.router.post(this.path, this.createEvent);
+    this.router.post(`${this.path}`, this.createEvent);
```

Comment:
I don't think this is required, unless we're using params (which are variable). Fix this.